client/my-sites/media-library/external-media-header.jsx

```diff
@@ -126,6 +166,7 @@ class MediaLibraryExternalHeader extends Component {
 					</Button>
 				) }
 
+				{ photosPickerApiEnabled && this.renderChangeSelectionButton() }
```

Comment:
Should we also check the source here so it only renders when the source is `google_photos`?

Reply:
Nice catch!!! I updated the condition.